### ISSUE TITLE
JSON Parsing issue in SimpleCookieCache

### ISSUE THREAD

**Issue by sirnornur** [WURFL/wurfl-cloud-client-java#28]
Dear project developers,

We use this library in our project and we noticed an issue since updating to v1.0.14.

The issue only occurs when the request contains the `WurflCloud_Client` cookie.

```
com.jsoniter.spi.JsonException: readString: expect string or null, but 1, head: 74, peek: ate_set":1, buf: {"ldate_set":1632314951,"capabilities":{"brand_name":"Robot"},"date_set":1632314951,"id":"deviceId"}
```

Prepared a short test case to reproduce the issue, uploading it as a file due to permission limits: [SimpleCookieCacheTest.zip](https://github.com/WURFL/wurfl-cloud-client-java/files/7210823/SimpleCookieCacheTest.zip)
The test file location: `wurfl-cloud-client-java/code/src/test/java/com/scientiamobile/wurflcloud/cache/SimpleCookieCacheTest.java`

I can confirm that v1.0.13 does not have this issue.
I hope this will help in preparing a bugfix.

Regards,
Siroj

**Comment by passani** [comment]
> The issue only occurs when the request contains the `WurflCloud_Client` cookie.
> 
 
Thank you for reporting this, Siroj

An engineer will address this issue next week after the release of our main product.

**Comment by andreacastello** [comment]
Hello @sirnornur a new version of the cloud client has been released (1.11.0) where the issue you opened has been fixed.
